### PR TITLE
init: task-2 Serve SPA in AWS

### PR DESCRIPTION
**Pull request by colpachoque**
Added and configured _serverless_ and _serverless-finch_ plugins
Automatically configured app was built and deployed by running npm script command `npm run cloudfront:update:build:deploy`
Links to the automatically configured app: 

- cloudfront link: https://dqx938xovkpeh.cloudfront.net/
- s3 link: http://colpachoque-first-app-1.s3-website-us-east-1.amazonaws.com/

Links to the manually configured app:

-  cloudfront link - https://d1qfdidd8o7j6.cloudfront.net
-  s3 link - http://colpachoque-dump-bucket.s3-website-eu-west-1.amazonaws.com/

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,3 +25,8 @@ yarn-error.log*
 
 /.idea
 .vscode
+
+jspm_packages
+
+# Serverless directories
+.serverless
```
Comment (NikaOrl) [inline .gitignore:31]: .serverless already exists in .gitignore

File: package.json [hunk 2]
```diff
@@ -6,33 +6,48 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "deploy:s3": "serverless client deploy --no-config-change --no-policy-change --no-cors-change",
```
Comment (NikaOrl) [inline package.json]: duplicates 'client:deploy' script

File: package.json [hunk 3]
```diff
@@ -6,33 +6,48 @@
...
+    "build:deploy": "npm run build && npm run deploy:s3",
```
Comment (NikaOrl) [inline package.json]: duplicates 'client:build:deploy' script

**Review by NikaOrl — CHANGES_REQUESTED** [review]
Please, follow convention for commit messages: https://docs.rs.school/#/git-convention and convention for PR description: https://docs.rs.school/#/stage2?id=Требования-к-pull-request-pr

Score: 5/5

File: package.json (hunk 2 above)
Comment (Ilya-Valasiuk) [reply package.json]: @NikaOrl I don't see any duplication. Could you please check again? :)

File: package.json (hunk 3 above)
Comment (Ilya-Valasiuk) [reply package.json]: the same here

File: package.json (hunk 2 above)
Comment (NikaOrl) [reply package.json]: @Ilya-Valasiuk `serverless client deploy --no-config-change --no-policy-change --no-cors-change` and `sls client deploy --no-config-change --no-policy-change --no-cors-change` are the same scripts

Comment (NikaOrl) [reply package.json]: It's not an error, just a recommendation so the score is still 5/5 from my point of view

File: package.json (hunk 3 above)
Comment (colpachoque) [reply package.json]: Thanks, guys!

**Pull request merged**